waiter/src/waiter/scheduler.clj

```diff
@@ -128,6 +128,11 @@
   (get-services [this]
     "Returns a list of scheduler/Service records")
 
+  (signal-instance [this instance signal-type] 
```

Comment:
Is the implementation simpler if the signature passes only instance-id and service-id?
Also, is there a need for passing it `timeout` as an argument?

```suggestion
  (signal-instance [this service-id instance-id signal-type timeout-ms] 
```